### PR TITLE
Update business-assist.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation as tenants with fewer than 5 users will be on simplified view

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5165

File: microsoft-365/admin/misc/business-assist.md [hunk 1]
```diff
@@ -46,9 +46,9 @@ Customers with an active Microsoft 365 for business subscription plan can use Bu
 
 Only organizations that are eligible will see the option to buy Business Assist. Business Assist costs $5 per user per month. Here's how you can buy Business Assist.
 
-1. In the Microsoft 365 admin center, go to **Billing** > **Purchase services**.
+1. In the simplified view of Microsoft 365 admin center, go to **Subscriptions** tab > **Add Products**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/misc/business-assist.md]:
```suggestion
1. In the simplified view of the Microsoft 365 admin center, go to the **Subscriptions** tab > **Add Products**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by JohanFreelancer9 — COMMENTED** [review]
Apologies @msbemba, one more, please.

File: microsoft-365/admin/misc/business-assist.md [hunk 2]
```diff
@@ -46,9 +46,9 @@ Customers with an active Microsoft 365 for business subscription plan can use Bu
 
 Only organizations that are eligible will see the option to buy Business Assist. Business Assist costs $5 per user per month. Here's how you can buy Business Assist.
 
-1. In the Microsoft 365 admin center, go to **Billing** > **Purchase services**.
+1. In the simplified view of the Microsoft 365 admin center, go to the **Subscriptions** tab > **Add Products**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/misc/business-assist.md]:
```suggestion
1. In the simplified view of the Microsoft 365 admin center, go to the **Subscriptions** tab and select **Add Products**.
```

File: microsoft-365/admin/misc/business-assist.md (hunk 1 above)
Comment (AntoniaBlume) [inline microsoft-365/admin/misc/business-assist.md]: Please change "Add Products" to "Add products" (lowercase)

File: microsoft-365/admin/misc/business-assist.md [hunk 3]
```diff
@@ -46,9 +46,9 @@ Customers with an active Microsoft 365 for business subscription plan can use Bu
 
 Only organizations that are eligible will see the option to buy Business Assist. Business Assist costs $5 per user per month. Here's how you can buy Business Assist.
 
-1. In the Microsoft 365 admin center, go to **Billing** > **Purchase services**.
+1. In the simplified view of Microsoft 365 admin center, go to **Subscriptions** tab > **Add Products**.
 
-2. Select **Details** on the **Small Business Assist for Microsoft 365** and complete your purchase.
+2. On the Purchase services page, Select **Details** on the **Business Assist for Microsoft 365** and complete your purchase.
```
Comment (AntoniaBlume) [inline microsoft-365/admin/misc/business-assist.md]: Please remove "the" before "Business Assist for Microsoft 365".

File: microsoft-365/admin/misc/business-assist.md [hunk 4]
```diff
@@ -22,7 +22,7 @@ description: "Learn about the Business Assist program and how it can help your o
 
 Get the most out of your subscription with expert advice from small business specialists.
 
-**Business Assist for Microsoft 365** is designed for businesses with less than 5 users to give you and your employees around-the-clock access to small business specialists as you grow your business, from onboarding to everyday use.
+**Business Assist for Microsoft 365** is designed for businesses with fewer than 5 users to give you and your employees around-the-clock access to small business specialists as you grow your business, from onboarding to everyday use.
```
Comment (AntoniaBlume) [inline microsoft-365/admin/misc/business-assist.md:25]: Please make this same change in the "Eligibility of Business Assist" section (change less to fewer)

**Review by AntoniaBlume — CHANGES_REQUESTED** [review]
See comments inline.

**Comment by msbemba** [comment]
> See comments inline.

Thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**